modules/samples/consumers/http-performance/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Thanks @prabathabey, changes are done in pull request - https://github.com/wso2/product-cep/pull/166